flexget/plugins/input/anilist.py

```diff
@@ -122,54 +122,65 @@ def on_task_input(self, task, config):
             try:
                 list_response = list_response.json()['data']
                 logger.debug('JSON output: {}', list_response)
-                for list_status in list_response['collection']['statuses']:
+                for list_status in list_response.get('collection', {}).get('statuses'):
                     if (
                         selected_list_name
-                        and list_status['name'].lower() not in selected_list_name
+                        and str(list_status.get('name')).lower() not in selected_list_name
```

Comment:
Are all of these fields really all that likely to contain something other than a string?

Reply:
They can be empty, in which case trying to perform a case operation like `lower()` results in a `NoneType` exception

Reply:
The string `"None"` doesn't seem like a good pick-- it looks too much like a real value (see: https://www.wired.com/story/null-license-plate-landed-one-hacker-ticket-hell/). Empty string would be better, or you can check for the presence of a value.